### PR TITLE
Cleanup SIGDescribe in test/e2e/network

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #98326

```release-note
NONE
```

/kind cleanup
/priority important-longterm

### CODE REVIEW

**Comment by aojea** [comment]
there are more sig-networks here
```
test/e2e/common/network/networking.go:var _ = SIGDescribe("Networking", func() {
test/e2e/common/network/framework.go:// SIGDescribe annotates the test with the SIG label.
test/e2e/common/network/framework.go:func SIGDescribe(text string, body func()) bool {
```

**Comment by wojtek-t** [comment]
> there are more sig-networks here

This is fine:
- test/e2e/common/network/networking.go:var _ = SIGDescribe("Networking", func() {

This is conscious decision (based on https://github.com/kubernetes/kubernetes/issues/98326) and we don't want to import it to test/e2e/common

- test/e2e/common/network/framework.go:

This is being removed in https://github.com/kubernetes/kubernetes/pull/99492

File: test/e2e/network/netpol/network_legacy.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-
+	"k8s.io/kubernetes/test/e2e/network/common"
```
Comment (aojea) [inline test/e2e/network/netpol/network_legacy.go:43]: what is wrong?
```
test/e2e/network/netpol/network_legacy.go:43:2: imported and not used: "k8s.io/kubernetes/test/e2e/network/common" as network
test/e2e/network/netpol/network_legacy.go:62:9: undefined: common
```

Comment (wojtek-t) [reply test/e2e/network/netpol/network_legacy.go:43]: This is super strange. But it seems the same was happening in instrumentation and they are importaint as instrumentation there, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/instrumentation/logging/generic_soak.go#L30

I will just change it - I'm not going to fight with compiler :)

**Comment by wojtek-t** [comment]
/hold

I will fix it but not today...

**Comment by aojea** [comment]
common maybe a forbidden word :)
```
test/e2e/network/conntrack.go:36:2: imported and not used: "k8s.io/kubernetes/test/e2e/network/common" as network
test/e2e/network/dns.go:32:2: imported and not used: "k8s.io/kubernetes/test/e2e/network/common" as network
test/e2e/network/dns_configmap.go:28:2: imported and not used: "k8s.io/kubernetes/test/e2e/network/common" as network
```

**Comment by wojtek-t** [comment]
/hold cancel

@aojea - the problem was that the added "framework.go" file had "package network". Fixed now, PTAL when you will have a chance.

**Comment by aojea** [comment]
/retest
/lgtm
:+1:

**Comment by Riaankl** [comment]
@wojtek-t @aojea I noticed that there are e2e tests for network policy covering the remaining test in the Network API group
https://apisnoop.cncf.io/1.21.0/stable/networking
The tests have a clean run on the [test grid ](https://k8s-testgrid.appspot.com/sig-release-master-blocking#gce-cos-master-default&include-filter-by-regex=should%20support%20creating%20NetworkPolicy%20API%20operations&width=5 ) and should be ready for Promotion to conformance.
Do you have plans to promote these? Or can I support you by creating the Promotion PR?

**Comment by aojea** [comment]
> Do you have plans to promote these? Or can I support you by creating the Promotion PR?

IIRC those were added by  @cmluciano , but to be honest I lost track on those, maybe he can expand

> Or can I support you by creating the Promotion PR?

that will be great , I can help reviewing

**Comment by Riaankl** [comment]
@aojea . great I will create the Promotion.
Just one question, the test exists in [network_legacy.go](https://github.com/kubernetes/kubernetes/blob/23d4b3b4f0d8b844f01b93262737c3d0efb6bff7/test/e2e/network/netpol/network_legacy.go#L2187-L2360)
And in 
[network_policy_api.go](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/netpol/network_policy_api.go#L39-L212)
It would not make sense to promote both.
Which one do you want me to promote?

**Comment by aojea** [comment]
> network_policy_api.go

this one ^^

**Pull request merged**